### PR TITLE
[DOC] RDoc for Complex

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: complex.c [hunk 1]
```diff
@@ -1850,16 +1853,11 @@ nucomp_to_r(VALUE self)
...
- * Returns the value as a rational if possible (the imaginary part
- * should be exactly zero).
+ * Equivalent to Complex#to_r.
  *
- *    Complex(1.0/3, 0).rationalize  #=> (1/3)
- *    Complex(1, 0.0).rationalize    # RangeError
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ * (Argument +epsilon+ is ignored.)
```
Comment (peterzhu2118) [inline complex.c:1860]: `epsilon` is not ignored. It's called with `self.real.rationalize(epsilon)`. For example:

```ruby
irb(main):004> Complex(1.3333, 0).rationalize
=> (13333/10000)
irb(main):005> Complex(1.3333, 0).rationalize(0.1)
=> (4/3)
```

File: complex.c [hunk 2]
```diff
@@ -1825,16 +1827,17 @@ nucomp_to_f(VALUE self)
 
 /*
  * call-seq:
- *    cmp.to_r  ->  rational
+ *   cmp.to_r  ->  rational
```
Comment (peterzhu2118) [inline complex.c:1830]:
```suggestion
 *  to_r  ->  rational
```

File: complex.c (hunk 1 above)
Comment (BurdetteLamar) [reply complex.c]: Revised.

File: complex.c (hunk 2 above)
Comment (BurdetteLamar) [reply complex.c]: Fixed.

File: complex.c [hunk 3]
```diff
@@ -1850,16 +1853,34 @@ nucomp_to_r(VALUE self)
...
- *    Complex(1, 0.0).rationalize    # RangeError
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ *   rationalize(epsilon = nil) -> rational
+ *
+ * Returns a Rational object whose value is exactly or approximately
+ * equivalent to that of <tt>self.real</tt>.
+ *
+ * With no argument +epsilon+ given, equivalent to Complex#to_r:
```
Comment (peterzhu2118) [inline complex.c:1861]: It's not equivalent to `Complex#to_r`, for example:

```ruby
irb(main):002> Complex(3.14159, 0).rationalize
=> (314159/100000)
irb(main):004> Complex(3.14159, 0).to_r
=> (3537115888337719/1125899906842624)
```

Comment (BurdetteLamar) [reply complex.c]: Fixed (I think).

File: complex.c [hunk 4]
```diff
@@ -1850,16 +1853,35 @@ nucomp_to_r(VALUE self)
...
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ *   rationalize(epsilon = nil) -> rational
+ *
+ * Returns a Rational object whose value is exactly or approximately
+ * equivalent to that of <tt>self.real</tt>.
+ *
+ * With no argument +epsilon+ given, returns a \Rational object
+ * whose value is exactly equal to that of <tt>self.real</tt>:
```
Comment (peterzhu2118) [inline complex.c:1862]: It calls the instance method `rationalize` on the real part.

```suggestion
 * whose value is exactly equal to that of <tt>self.real.rationalize</tt>:
```

**Review by peterzhu2118 — APPROVED** [review]

**Pull request merged**